doc/itof/nxp_build_instructions.md

```diff
@@ -99,6 +101,8 @@ Please note, ensure you are using the intended branch.
 ```console
 git clone --branch v5.0.0  https://github.com/analogdevicesinc/ToF
 cd ToF
+git submodule ini
+git submodule update
```

Comment:
You can reduce to one line:
git submodule update --init

Reply:
Changed